.ci/bears.prune.sh

```diff
@@ -0,0 +1,7 @@
+# Delete empty bear directories
+
+for dir in bears/*/; do
+  if [[ -z "$(ls $dir/ 2>/dev/null | egrep -v '(__|.xml)')" ]]; then
```

Comment:
Oh, the problem with *Bear is we have directories with only subdirectories, like `bears/vcs`.  Solvable, and probably still clearer to take this approach.